core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -844,4 +851,72 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
     assertEquals(2, rows.size)
     rows(0).startsWith(s"Topic:$testTopicName\tPartitionCount:1")
   }
+
+  @Test
+  def testCreateTopicDoesNotRetryThrottlingQuotaExceededException(): Unit = {
+    val adminClient = Mockito.mock(classOf[Admin])
+    val topicService = AdminClientTopicService(adminClient)
+
+    val result = AdminClientTestUtils.createTopicsResult(testTopicName, Errors.THROTTLING_QUOTA_EXCEEDED.exception())
+    Mockito.when(adminClient.createTopics(ArgumentMatchers.any(), ArgumentMatchers.any())).thenReturn(result)
```

Comment:
May be worth importing `org.mockito.Mockito._` and  `org.mockito.ArgumentMatchers._` to avoid repeating the class name everywhere.

Reply:
That makes sense. I had to introduce an alias for `ArgumentMatchers.eq` to not conflict with `eq`. I went with `eqThat` to remain inline with `argThat`.